### PR TITLE
fix: add details of all speakers to new session mail

### LINKED ISSUE #8486: New session email to organizers uses CfS link and needs text improvements
The "new session" email to organizers uses the CfS link of the session which is used for speakers. When changing this email, please also enhance the email. So expected is: It should use the organizer session link.

1. Current email and link:
Hi office@fossasia.org,
The event FOSSASIA Summit 2021 has received a new session proposal.
Visit this link to view the session: https://eventyay.com/e/fa96ae2c/cfs/session/6760

2. Correct link would be: https://eventyay.com/events/fa96ae2c/session/6760
3. Change email text as follows:

Subject: New session proposal for the {event_name} titled "{session_title}".

Hello,

This is an automatic email from {app name} for organizers of the "{event_name}" [add link to public event page].

Your Call for Speakers [add link to CfS page] has received a new session proposal with the following details:

Speaker: {speaker_name}
Email: {speaker_email}
Bio: {speaker_bio}
Session: {session_title}
Abstract: {session_abstract}

You can view all speaker details on the speaker profile [add link to profile for organizers to "speaker profile"] page and the session proposal on the session page [add link to "session page"]. 

To change the status of sessions please go to the sessions overview page [add link to sessions overview page] and click on the status badge of the session.

Thank you.
{app name frontpage}

4. Ensure everything is translatable.

### PR DESCRIPTION
**Pull request by pc-beast**
Fixes #8486 

image

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Review by progmatic-99 — COMMENTED** [review]
I am getting **two mails**. In one, there is speaker details and in other, there is no speaker details.

The cfs link is not changed.

File: app/api/sessions.py [hunk 1]
```diff
@@ -86,6 +81,14 @@ def after_create_object(self, session, data, view_kwargs):
                 speaker_id=speaker.id,
             )
             save_to_db(session_speaker_link, "Session Speaker Link Saved")
+        
+        if session.event.get_owner():
+            owner = session.event.get_owner()
+            owner_email = owner.email
+            if session.speakers:
+                print("yes")
```
Comment (progmatic-99) [inline app/api/sessions.py:89]: This is not getting printed.

File: app/api/sessions.py [hunk 2]
```diff
@@ -86,6 +81,13 @@ def after_create_object(self, session, data, view_kwargs):
                 speaker_id=speaker.id,
             )
             save_to_db(session_speaker_link, "Session Speaker Link Saved")
+        
+        if session.event.get_owner():
```
Comment (progmatic-99) [inline app/api/sessions.py:85]: Can we remove these lines of code from here ?

**Review by progmatic-99 — APPROVED** [review]
Rest works fine!!

File: app/api/sessions.py (hunk 2 above)
Comment (pc-beast) [reply app/api/sessions.py:85]: No, they are invoked when organiser creates session from event dashboard and adds the speaker from there.

**Pull request merged**